components/atoms/InsightBadge/insight-badge.tsx

```diff
@@ -0,0 +1,17 @@
+import Text from "components/atoms/Typography/text";;
+
+interface BadgeProps {
+    isPublic: boolean
+}
+
+const Badge = ({ isPublic }: BadgeProps): JSX.Element => {
+  return (
+    <div className="inline-flex border border-light-slate-6 rounded-[26px] bg-light-slate-3 px-2.5 py-1">
+      <span className="text-sm text-light-slate-11 leading-none">
```

Comment:
replacing the `Text` component with a span because it adds unnecessary line height that breaks the badge
cc: @bdougie @0-vortex

Reply:
Thanks for moving this to its component and add the story

Reply:
sure, i went through the readme like you suggested 😅